AdblockPlusSafari/AdblockPlusContainerController.m

```diff
@@ -53,6 +53,18 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender
   if ([segue.destinationViewController respondsToSelector:@selector(setAdblockPlus:)]) {
     [(id)segue.destinationViewController setAdblockPlus:self.adblockPlus];
   }
+
+#ifdef CONFIGURABLE_CUSTOM_FILTER_LIST_ENABLED
+  if ([segue.identifier isEqualToString:@"ConfigureFilterListsSegue"]) {
```

Comment:
There is request for unusual name for back button in navigation bar of custom filter lists controller.